src/cli/commands/list-tests/index.ts

```diff
@@ -0,0 +1,64 @@
+import path from "node:path";
+import fs from "fs-extra";
+import { ValueOf } from "type-fest";
```

Comment:
type-fest is currently a dev dependency. we already experienced issues because of it (when you install testplane, but do not have type-fest which is needed to correctly resolve all typings). I suggest to either inline ValueOf type or move type-fest to dependencies.